open-media-match/src/OpenMediaMatch/migrations/versions/76896c08b4ff_.py

```diff
@@ -0,0 +1,39 @@
+"""empty message
```

Comment:
Doing migrations at this stage seems like an undue burden. Easy enough to do `flask reset_all_tables`